apps/www/app/api-v2/opt-out/[ref]/route.ts

```diff
@@ -78,7 +79,10 @@ export async function POST(req: NextRequest, props: { params: Promise<{ ref: str
       .from('manual_reports')
       .insert([{ project_ref: ref, reason, email }])
 
-    if (supabaseError) throw new Error(`Supabase error: ${supabaseError.message}`)
+    if (supabaseError) {
+      Sentry.captureException(supabaseError)
```

Comment:
I think this might be double-sending the error, since it's caught below and sent again

Reply:
oh great catch! sorry hahaha